### PR TITLE
Improve docs & update plugin list

### PR DESCRIPTION
**Pull request by DavidWells**
## What did you implement:

Readme data gets stale quickly. I've implemented a script to keep plugins up to date with the latest plugin data from the https://github.com/serverless/community-plugins/blob/master/plugins.json repo.

The script will grab the remote data, and format it however we want, and insert it back into the `README.md` file.

To run the script run `node scripts/generate-readme.js`. This should probably be in the `npm scripts`

This documentation setup can also be used for keeping other markdown files in sync with code or other remote resources.

### CODE REVIEW

**Comment by nikgraf** [comment]
This is fantastic! 🙌 

I propose we run this script as a PrePublish step. This way NPM always has the latest and we regularly regenerate it. Of course it can be regenerated manually more often. Thoughts @DavidWells

**Comment by DavidWells** [comment]
@pmuens can you merge this? CI/CD is failing on .md changes again =(

**Comment by DavidWells** [comment]
@pmuens nvm. node 4 errored on `let =P`

File: package.json [hunk 1]
```diff
@@ -49,6 +49,8 @@
   "scripts": {
     "test": "istanbul cover -x '**/*.test.js' node_modules/mocha/bin/_mocha '!(node_modules)/**/*.test.js' -- -R spec --recursive",
     "lint": "eslint .",
+    "docs": "node scripts/generate-readme.js",
+    "prepublish": "npm run docs",
```
Comment (pmuens) [inline package.json]: ❤️

Comment (nikgraf) [inline package.json]: thx @DavidWells - automate everything 👍

**Pull request merged**